docs/_user_guide/best_practices_matlab_slurm_job.md

```diff
@@ -1,33 +1,33 @@
 ## Matlab Example Slurm Script
 The simplest Matlab script looks like:
 
-```matlab
+```matlab title="Matlab Code"
 disp('Hello world')
 ```
 
 Save this line to a new file called `hello_world.m`. 
 
-## Available Matlab versions on Yen10
+## Available Matlab versions on `Yen10`
 Several version of Matlab are installed on the yens and the default version is designated with `(D)`:
 
-```bash
+```bash title="Terminal Command"
 module avail matlab
 ```
 
 Load the version you want:
 
-```bash
+```bash title="Terminal Command"
 module load matlab
 ```
 
 If you do not specify the version, `matlab/R2019b` will be loaded by default.
 
 This one-liner script can be run with `matlab -nodesktop < hello_world.m`. 
-However, we will run it via the Slurm scheduler on the Yen10 server. 
-Here is an example Slurm script that loads matlab module and runs hello world script.
+However, we will run it via the Slurm scheduler on the `Yen10` server. 
```

Comment:
However, if you want to run it on the [Yen Slurm cluster](/_user_guide/slurm/){:target="_blank"} via the scheduler, you must write a Slurm script.